frontend/pages/policies/PolicyPage/components/PolicyForm/PolicyForm.tsx

```diff
@@ -52,7 +54,7 @@ const PolicyForm = ({
   onOpenSchemaSidebar,
   renderLiveQueryWarning,
 }: IPolicyFormProps): JSX.Element => {
-  const [errors, setErrors] = useState<{ [key: string]: string }>({});
+  const [errors, setErrors] = useState<{ [key: string]: any }>({});
```

Comment:
I think maybe we could use `unknown` instead of `any` in cases like this? It's something I just learned about so I don't know all the specifics but it looks promising. https://stackoverflow.com/questions/51439843/unknown-vs-any